PokemonRampUp/Model/Network.swift

```diff
@@ -29,6 +29,7 @@ struct DefaultNetwork: Network {
          }
 
          let decoder = JSONDecoder()
+         decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Need to add this to the Network file because the decoder needs to convert JSON variable/object names to snake case